src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -188,8 +188,12 @@ class DefaultSource
         case _ => sys.error(s"Resource type $resourceType does not support save()")
       }
       val batchSizeDefault = relation match {
-        case _: NumericDataPointsRelationV1 => Constants.CreateDataPointsLimit
-        case _: StringDataPointsRelationV1 => Constants.CreateDataPointsLimit
+        case _: NumericDataPointsRelationV1 | _: StringDataPointsRelationV1 =>
+          if (config.onConflict == OnConflict.Delete) {
+            Constants.DefaultDataPointsLimit
+          } else {
+            Constants.CreateDataPointsLimit
+          }
```

Comment:
why

Reply:
The datapoint endpoint allows 100_000 items when creating, but only 10_000 when deleting

Reply:
can you add a comment on this?

Reply:
in case someone wonders why in the future